From 7693fc35721ea7e1c39b37b9b66a134b65720ef8 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Tue, 25 Jun 2024 17:47:05 +0800
Subject: [PATCH 56/77] arm64: dts: rockchip: Add wifi regulator for Cool Pi 4b

This will brocken spi nor if this dts sync to u-boot

Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3588s-coolpi-4b.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588s-coolpi-4b.dts b/arch/arm64/boot/dts/rockchip/rk3588s-coolpi-4b.dts
index d938db0e2239c..25de2c1ef4136 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588s-coolpi-4b.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588s-coolpi-4b.dts
@@ -172,6 +172,18 @@ vcc5v0_otg: vcc5v0-otg-regulator {
 		vin-supply = <&vcc5v0_sys>;
 	};
 
+	vcc1v8_wifi: vcc1v8-wifi-regulator {
+		compatible = "regulator-fixed";
+		enable-active-high;
+		gpio = <&gpio1 RK_PD3 GPIO_ACTIVE_HIGH>;
+		regulator-name = "vcc3v3_wifi";
+		regulator-boot-on;
+		regulator-always-on;
+		regulator-min-microvolt = <1800000>;
+		regulator-max-microvolt = <1800000>;
+		vin-supply = <&vcc5v0_sys>;
+	};
+
 	vcc_1v1_nldo_s3: vcc-1v1-nldo-s3-regulator {
 		compatible = "regulator-fixed";
 		regulator-name = "vcc_1v1_nldo_s3";
-- 
2.47.0

